packages/manager/modules/core/src/redirection/redirection.constants.js

```diff
@@ -21,6 +21,7 @@ export default {
     paymentMethod: `${managerRoot.EU}/manager/dedicated/${URI.paymentMethod}`,
     services: `${managerRoot.EU}/manager/dedicated/${URI.services}`,
     support: `${managerRoot.EU}/manager/dedicated/${URI.support}`,
+    ticket: `${managerRoot.EU}/manager/dedicated/#/support/tickets/:ticketId`,
```

Comment:
Looks like you could abstract the `/manager/dedicated/` part too, can't you ?

ie

```js
${eu.manager.dedicated}/${URI.support}
```

for example

Reply:
the problem is that it's just ```/manager``` for CA region

Reply:
`${eu.manager.dedicated}`=== `https://www.ovh.com/manager/dedicated`

and

`${ca.manager.dedicated}`=== `https://www.ovh.com/manager`

Is that an issue ?

Reply:
yep refactoring will be done in another PR